src/nanoemoji/config.py

```diff
@@ -288,7 +288,8 @@ def write(dest: Path, config: FontConfig):
             for m in config.masters
         },
     }
-    dest.write_text(toml.dumps(toml_cfg))
+    toml_cfg = {key: val for key, val in toml_cfg.items() if val is not None}
```

Comment:
why is None special now? is that a know difference between the old and new toml libraries?

Reply:
Yes, tomlkit raises an error on None values: https://github.com/sdispater/tomlkit/issues/240

Reply:
Added a comment for that workaround